Scripts/BuildDatabank/searchDATABANK.py

```diff
@@ -1,9 +1,10 @@
-import os
+import os, sys
 import yaml
 import json
 
-from databankLibrary import lipids_dict
 from tqdm import tqdm
+sys.path.append('..')
+from DatabankLib.databankLibrary import lipids_dict
 
 databank_path = '../../Data/Simulations'
 expbank_path = '../../Data/experiments'
```

Comment:
both `databank_path` and `expbank_path`  occur in multiple scripts (sometimes even with different names, see `makeRanking.py`). 

we should make these imported/read from a separate file (something like `config.json`) which contains the default variables in the code. we can work on this separately or you can prepare a new pull requests addressing this. i leave the decision to you

Reply:
explicit path were removed in upcoming commits